.github/workflows/jira.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Export issue to Jira
+
+on:
+  issues:
+    types: [labeled]
+
+permissions:
+  issues: write
+
+jobs:
+  export:
+    uses: "puppetlabs/phoenix-github-actions/.github/workflows/jira.yml@main"
+    with:
+      jira-project: PA
```

Comment:
Note this creates issues in the PA project, but doesn't apply the `cron_core` component